### PR TITLE
Allow import image into empty imagestream

### PR DESCRIPTION
**Pull request by legionus**
Fixes https://github.com/openshift/origin/issues/14631

### CODE REVIEW

File: pkg/oc/cli/cmd/importimage.go [hunk 1]
```diff
@@ -359,11 +359,6 @@ func (o *ImportImageOptions) createImageImport() (*imageapi.ImageStream, *imagea
 		return stream, isi, nil
 	}
 
-	// the stream already exists
-	if len(stream.Spec.DockerImageRepository) == 0 && len(stream.Spec.Tags) == 0 {
-		return nil, nil, fmt.Errorf("image stream does not have valid docker images to be imported")
-	}
-
```
Comment (bparees) [inline pkg/oc/cli/cmd/importimage.go]: throwing this error seems correct. I'm confused by the issue the users opened, need to get some clarification on what they are expecting and why.

Comment (legionus) [reply pkg/oc/cli/cmd/importimage.go]: @bparees Empty imagestreams can exist. This is not something wrong. You can remove all imagestream tags from imagestream and import new tags.

The `stream.Spec.DockerImageRepository` and `stream.Spec.Tags` are processed later in the `importAll` and `importTag`. So this check looks completely wrong for me.

**Pull request merged**